batch/batch/driver/instance_pool.py

```diff
@@ -153,8 +153,15 @@ def add_instance(self, instance):
             if machine_name not in self.name_instance:
                 break
 
-        zone_letter = random.choice(['a', 'b', 'c', 'f'])
-        zone = f'us-central1-{zone_letter}'
+        if self.live_free_cores_mcpu < 50_000_000:
+            zones = ['us-central1-a', 'us-central1-b', 'us-central1-c', 'us-central1-f']
+            zone = random.choice(zones)
+        else:
+            zones = ['us-central1-a', 'us-central1-b', 'us-central1-c', 'us-central1-f', 'us-east1-a', 'us-east1-b', 'us-east1-c', 'us-east4-a', 'us-east4-b', 'us-east4-c', 'us-west1-a', 'us-west1-b', 'us-west1-c', 'us-west2-a', 'us-west2-b', 'us-west2-c']
+            # based on quotas, us-central1: 300K over 4 zones, rest: 100K over 3 zones
```

Comment:
This comment should either be 250 or the line below should be 300.

Reply:
I subtracted the 50K from above.